backend/python/app/tests/services/test_story_service.py

```diff
@@ -11,23 +14,51 @@ def test_get_story(app, db, services):
     assert_story_equals_model(resp, obj, graphql_response=False)
 
 
-def test_get_story_invalid_id():
-    pass
-
-
-def test_get_stories():
-    pass
+def test_get_story_invalid_id(app, db, services):
+    with pytest.raises(Exception) as e:
+        _ = services["story"].get_story(-1)
+        assert "Invalid id" in str(e.value)
 
+    obj = Story(title="title", description="description", youtube_link="", level=1)
+    db.session.add(obj)
+    assert db.session.commit() == None
 
-def test_create_story():
-    pass
+    with pytest.raises(Exception) as e:
+        _ = services["story"].get_story(obj.id + 1)
+        assert "Invalid id" in str(e.value)
 
 
-def test_create_story_raises_error_if_story_commit_fails_and_nothing_saved():
-    pass
+def test_get_stories(app, db, services):
+    obj = Story(title="title", description="description", youtube_link="", level=1)
+    db.session.add(obj)
+    assert db.session.commit() == None
 
+    obj_1 = Story(
+        title="title_1", description="description_1", youtube_link="", level=2
+    )
+    db.session.add(obj_1)
+    assert db.session.commit() == None
 
-def test_create_story_raises_error_if_story_content_commit_fails_and_nothing_saved():
+    resp = services["story"].get_stories()
+    assert_story_equals_model(resp[0], obj, graphql_response=False)
+    assert_story_equals_model(resp[1], obj_1, graphql_response=False)
+
+
+def test_create_story(app, db, services):
+    story = StoryRequestDTO(
```

Comment:
couldnt use the `StoryRequestDTO` from `..graphql.types.story_types` because `graphene.Object` is a weird class and throws errors at `new_story = Story(**story.__dict__)` in the service :(